server/config/config.js

```diff
@@ -11,7 +11,15 @@ const config = {
     username: process.env.POSTGRES_USERNAME,
     password: process.env.POSTGRES_PASSWORD,
     database: 'mine_claims',
-    host: 'mine-support-postgres-claims',
+    host: 'localhost',
+    port: 5432,
+    dialect: 'postgres'
+  },
+  test: {

```

Comment:
I that's only at the moment though. It could quite easily change in the future when we do more config on the dev and test environments. Worth leaving for now I think.

Reply:
Or would you control the differences by env vars, and have them all be the same?
Fine to leave as is for now.